compiler/rustc_builtin_macros/src/deriving/encodable.rs

```diff
@@ -5,14 +5,14 @@
 //!
 //! For example, a type like:
 //!
-//! ```
+//! ```ignore HELP (it says RustcEncodable and RustcEncodable are missing generic parameters? also it says these are deprecated and I can't find any instances of them outside these files. can we remove the whole thing?)
```

Comment:
They are deprecated but stable and thus can't be removed. They need to be used in combination with the rustc-serialize crate on crates.io (a precursor to serde). Note that compiler/rustc_serialize in this repo has diverged significantly from what RustcEncodable and RustcDecodable need.

Reply:
Oh I didn't realize there was code in `compiler/` that made stability guarantees to code outside the rust repo.
So I should put the crates.io `rustc-serialize` in the [dev-dependencies]`?

Reply:
I think ignoring it is fine. There are already some tests for it in `src/test` anyway.